src/Command/GarbageCollector/AssignmentGarbageCollectorCommand.php

```diff
@@ -138,8 +139,14 @@ private function collectStuckAssignments(int $batchSize, bool $isDryRun): int
                 $batchSize
             );
 
+            /** @var ArrayIterator $stuckAssignmentsIterator */
+            $stuckAssignmentsIterator = $stuckAssignments->getIterator();
+            $assignmentCount = $stuckAssignmentsIterator->count();
+
+            $logMessagePlaceholder =
```

Comment:
Not sure we could/should break it like this. Anyway, not real gain from previous place

Reply:
Fixed